react/src/components/ImageList.tsx

```diff
@@ -0,0 +1,512 @@
+        }}
+      >
+        <Flex justify="end" style={{ padding: token.paddingSM }} gap={'xs'}>
+          {selectedRows.length > 0 ? (
+            <Typography.Text>
+              {t('general.NSelected', {
+                count: selectedRows.length,
+              })}
+            </Typography.Text>
+          ) : null}
+          <Input
+            allowClear
+            prefix={<SearchOutlined />}
+            placeholder={t('environment.SearchImages')}
+            onChange={(e) => {
+              startSearchTransition(() => setImageSearch(e.target.value));
+            }}
+            style={{
+              width: 200,
+            }}
+          />
+          <Button
+            icon={<ReloadOutlined />}
+            loading={isPendingRefreshTransition}
+            onClick={() => {
+              setSelectedRows([]);
+              startRefreshTransition(() => updateEnvironmentFetchKey());
+            }}
+          >
+            {t('button.Refresh')}
+          </Button>
+
+          <Button
+            icon={<VerticalAlignBottomOutlined />}
+            style={{ backgroundColor: token.colorPrimary, color: 'white' }}
+            onClick={() => {
+              if (selectedRows.length === 0) {
+                message.error(t('environment.NoImagesAreSelected'));
+                return;
+              }
+              if (selectedRows.some((image) => !image.installed)) {
+                setIsOpenInstallModal(true);
+                return;
+              }
+              message.error(t('environment.AlreadyInstalledImage'));
+            }}
+          >
+            {t('environment.Install')}
+          </Button>
+        </Flex>
+        <Table<EnvironmentImage>
+          rowKey="id"
+          scroll={{ x: 'max-content' }}
+          pagination={{
+            showTotal(total, range) {
+              return `${range[0]}-${range[1]} of ${total} items`;
+            },
+            pageSizeOptions: ['10', '20', '50'],
+            style: { marginRight: token.marginXS },
+          }}
+          dataSource={filterNonNullItems(filteredImageData)}
+          columns={columns}
+          loading={isPendingSearchTransition}
+          rowSelection={{
+            type: 'checkbox',
+            // hideSelectAll: true,
+            // columnWidth: 48,
+            onChange: (_, selectedRows: EnvironmentImage[]) => {
+              setSelectedRows(selectedRows);
+            },
+            selectedRowKeys: selectedRows.map((row) => row.id) as Key[],
+          }}
+          onRow={(record) => ({
+            onClick: (e) => {
+              // selected or deselect row
+              if (selectedRows.find((row) => row.id === record.id)) {
+                setSelectedRows((rows) =>
+                  rows.filter((row) => row.id !== record.id),
+                );
+              } else {
+                setSelectedRows((rows) => [...rows, record]);
+              }
+            },
+          })}
+          showSorterTooltip={false}
```

Comment:
```suggestion
          showSorterTooltip={false}
          sortDirections={['descend', 'ascend', 'descend']}
```
Logic to limit data sorting to ascending and descending order is not needed in the webui?